bin/auto-update/index.ts

```diff
@@ -27,6 +29,8 @@ const settings: Settings = {
   dtRepoOwner: 'test-user-delete-me-please-3', // TODO: change this to: 'DefinitelyTyped'
   dtRepoName: 'DefinitelyTyped',
   thisRepo: 'google-api-typings-generator',
+  pullRequestTemplateSHA: 'ab3f84d44f18d1f33cc79c5d857067b939813c7e',
+  templateUpdateLabel: 'template update',
```

Comment:
Very minor, but thoughts on renaming this label to `definitely typed`? That might clarify this relates to this DT automation as opposed to another "template" concept in the generator source code.

Reply:
Good point, I renamed it to `DT PR template update`, also added a description that can be seen by hovering over the label.